controller/src/main/java/io/pravega/controller/server/SegmentStoreConnectionManager.java

```diff
@@ -303,6 +303,11 @@ public void segmentRead(WireCommands.SegmentRead segmentRead) {
             execute(ReplyProcessor::segmentRead, segmentRead);
         }
 
+        @Override
+        public void flushedStorage(WireCommands.FlushedStorage flushedStorage) {
```

Comment:
Why is the Controller required to know anything about this admin command? This does not look correct.